src/DocumentFormat.OpenXml.Linq/DocumentFormat.OpenXml.Linq.csproj

```diff
@@ -0,0 +1,51 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), DocumentFormat.OpenXml.Package.props))\DocumentFormat.OpenXml.Package.props" />
+
+  <PropertyGroup>
+    <Nullable>enable</Nullable>
+    <IsShipped>true</IsShipped>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```

Comment:
For now, set this to `$(ProductTargetFrameworks)`. The build ends up building a matrix of various permutations and sometimes the output is just .NET 3.5 or .NET 4.0, which would not be consumable here